neuromation/api/core.py

```diff
@@ -80,14 +83,28 @@ def __init__(
         }
         self._prev_cookie: Optional[Morsel[str]] = None
 
-    def _post_init(self, db: sqlite3.Connection, storage_url: URL) -> None:
-        cookie_val = load_cookie(db)
+    def _post_init(
+        self,
+        db: sqlite3.Connection,
+        storage_url: URL,
+        registry_url: URL,
+        blob_storage_url: URL,
+    ) -> None:
+        for url in (storage_url, registry_url, blob_storage_url):
+            if url is not None:
+                domain = url.host
+                path = url.path
+                if domain is not None:
+                    self._load_cookie(db, domain, path)
```

Comment:
won't this be overriding the previously set values?

Reply:
outdated